backend/hasura-actions/src/Scheduler.res

```diff
@@ -27,67 +27,30 @@ let makePayment = (~recipientAddress, ~paymentData: streamData) => {
-  PaymentStreamManager.gqlClient.mutate(
-    ~mutation=module(Query.AddNewPayment),
-    Query.AddNewPayment.makeVariables(
-      ~streamID=paymentData.streamID,
-      ~paymentTimestamp=paymentData.currentPayment->BN.toNumber,
-      ~paymentState="PENDING",
-      ~paymentAmount=finalAmount->BN.toString,
-      (),
-    ),
+  PaymentStreamManager.addPaymentEntry(
+    ~streamID=paymentData.streamID,
+    ~timestamp=paymentData.nextPayment->BN.toNumber,
+    ~amount=finalAmount->BN.toString,
   )
-  ->JsPromise.map(result =>
-    switch result {
-    | Ok({data: insert_payments_one}) =>
-      ()
-
-      Js.log("success payment added")
-    | Error(error) => Js.log2("error payment added: ", error)
-    }
-  )
-  ->ignore
-  if BN.eq(paymentData.numberOfPayments, BN.add(paymentData.numberOfPaymentsMade, finalPayment)) {
-    PaymentStreamManager.gqlClient.mutate(
-      ~mutation=module(Query.CloseStreamEntry),
-      Query.CloseStreamEntry.makeVariables(
-        ~id=paymentData.streamID,
-        ~paymentsMade=paymentData.numberOfPayments->BN.toNumber,
-        ~state="CLOSED",
-        (),
-      ),
-    )
-    ->JsPromise.map(result =>
-      switch result {
-      | Ok(_result) => Js.log("success close entry: CLOSED")
-      | Error(error) => Js.log2("error close entry: ", error)
-      }
+  let totalPayments = BN.add(paymentData.numberOfPaymentsMade, finalPayment)
+  if BN.eq(paymentData.numberOfPayments, totalPayments) {
+    PaymentStreamManager.closeStreamEntry(
+      ~streamID=paymentData.streamID,
+      ~totalPaymentsMade=paymentData.numberOfPayments->BN.toNumber,
     )
-    ->ignore
   } else {
     let newPaymentsMade = BN.add(paymentData.numberOfPaymentsMade, finalPayment)
     let intervalInSeconds = BN.mul(paymentData.interval, CONSTANTS.big60)
     let newNextPayment = BN.add(paymentData.nextPayment, BN.mul(finalPayment, intervalInSeconds))
-    PaymentStreamManager.gqlClient.mutate(
-      ~mutation=module(Query.UpdateStreamEntry),
-      Query.UpdateStreamEntry.makeVariables(
-        ~id=paymentData.streamID,
-        ~paymentsMade=newPaymentsMade->BN.toNumber,
-        ~nextPayment=newNextPayment->BN.toNumber,
-        ~lastPayment=paymentData.nextPayment->BN.toNumber,
-        (),
-      ),
-    )
-    ->JsPromise.map(result =>
-      switch result {
-      | Ok(_result) => Js.log3("success payment made: ", newPaymentsMade, newNextPayment)
-      | Error(error) => Js.log2("error payment made: ", error)
-      }
+    PaymentStreamManager.updateStreamEntry(
+      ~streamID=paymentData.streamID,
+      ~totalPaymentsMade=newPaymentsMade->BN.toNumber,
+      ~nextPayment=newNextPayment->BN.toNumber,
+      ~lastPayment=paymentData.nextPayment->BN.toNumber,
     )
-    ->ignore
   }
-  /* let requestString =
-    "http://localhost:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/" ++
+  let requestString =
+    "http://raiden1:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/" ++
```

Comment:
This address shouldn't be hard-coded. It will be different for different hubs...

Quick win could be to make it an environment variable? 

#32